src/coreclr/jit/compiler.cpp

```diff
@@ -5106,9 +5104,7 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
     compJitTelemetry.NotifyEndOfCompilation();
 #endif
 
-#if defined(DEBUG)
-    ++Compiler::jitTotalMethodCompiled;
-#endif // defined(DEBUG)
+    InterlockedIncrement(&Compiler::jitTotalMethodCompiled);
```

Comment:
Store the value returned by `InterlockedIncrement` and use it for the logging statement later?

Reply:
Thanks, done